package.json

```diff
@@ -84,33 +84,33 @@
   "devDependencies": {
     "assert": "^2.0.0",
     "eslint": "^6.1.0",
-    "mocha": "^6.2.0",
+    "mocha": "^6.2.3",
     "mocha-junit-reporter": "^1.23.0",
     "mocha-multi-reporters": "^1.1.7",
     "nyc": "^14.1.1",
     "supertest": "^4.0.2",
     "validator": "^11.1.0"
   },
   "dependencies": {
-    "antlr4": "^4.7.2",
+    "antlr4": "^4.9.2",
     "applicationinsights": "^1.4.1",
     "body-parser": "^1.19.0",
-    "codemirror": "^5.46.0",
+    "codemirror": "^5.62.0",
     "cookie-parser": "^1.4.4",
     "deepmerge": "^4.2.2",
     "express": "^4.16.4",
     "fhir": "^4.7.9",
     "fs-extra": "^8.1.0",
-    "handlebars": "^4.7.6",
-    "lodash": "^4.17.19",
+    "handlebars": "^4.7.7",
+    "lodash": "^4.17.21",
     "memory-cache": "^0.2.0",
     "ncp": "^2.0.0",
-    "nodegit": "^0.24.3",
+    "nodegit": "^0.27.0",
```

Comment:
We can just remove all the nodegit related snippets since we are not using the git mechanism any more. Previously we just hided the entrance of git functions, now we can thoroughly remove the git functions.

Reply:
I will check with @qiwjin: if we remove nodegit in this pr or in another pr?

Reply:
Whether to remove the nodegit or not is not an emergent thing, I think we can remove it later instead of in this pull request.

Reply:
I will remove it in another pr.